pkg/api/validate/plugin.go

```diff
@@ -90,6 +91,16 @@ func (v *PluginAPIValidator) Validate(c *pluginapi.Config) (errs []error) {
 	return
 }
 
+func validateSecurityPatchPackages(names []string) (errs []error) {
```

Comment:
i hate named return values, but that's a personal pref only :man_shrugging:

Reply:
I just followed the function naming convention in that file. If others feel as strongly about this as you do then we could  discuss it and cook it up in a follow-up. What do you think?